Dockerfile.cli.release

```diff
@@ -1,9 +1,30 @@
 # we use alpine for easier debugging
 FROM alpine:3.18
 
+ARG HELM_VERSION="v3.11.0"
```

Comment:
Could you please align this with the helm version in the vCluster Dockerfile?

iirc that one is using a newer version of helm